src/libs/OptionsListUtils.ts

```diff
@@ -725,7 +725,7 @@ function createOption(
 /**
  * Get the option for a policy expense report.
  */
-function getPolicyExpenseReportOption(report: Report): ReportUtils.OptionData {
+function getPolicyExpenseReportOption(report: Partial<Report>): ReportUtils.OptionData {
```

Comment:
Why this change?

Reply:
<img width="690" alt="Screenshot 2024-02-09 at 16 19 10" src="https://github.com/Expensify/App/assets/48553277/108a94a0-d279-4051-99f6-44165829e421">
@fabioh8010

Reply:
I ended up reverting it and I just manually pass the reportID